File: 백준/Bronze/2309.일곱 난쟁이/일곱 난쟁이.java
import java.util.*;
import java.io.*;


public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        ArrayList<Integer> list = new ArrayList<>();
        int total = 0;
        for (int i = 0; i < 9; i++) {
            int n = Integer.parseInt(br.readLine());
            list.add(n);
            total += n;
        }
        int over = total - 100;
        boolean flag = false;
        int a = 0;
        int b = 0;
        for (int i = 0; i < list.size() - 1; i++) {
            for (int j = i + 1; j < list.size(); j++) {
                if (list.get(i) + list.get(j) == over) {
                    a = list.get(i);
                    b = list.get(j);
                    flag = true;
                }
                if (flag) break;

                if (flag) break;
            }
            if (flag) break;
        }
        list.remove(list.indexOf(a));
        list.remove(list.indexOf(b));

        Collections.sort(list);
        for (Integer i : list) {
            System.out.println(i);
        }


    }
}

File: 백준/Bronze/2309. 일곱 난쟁이/일곱 난쟁이.py
import sys

array = []
n = 9
for _ in range(n):
    array.append(int(sys.stdin.readline()))


total = sum(array)
flag = True
for i in range(n-1):
    if not flag:
        break

    for j in range(i + 1, n):
        if total - (array[i] + array[j]) == 100:
            n1 = array[i]
            n2 = array[j]
            array.remove(n1)
            array.remove(n2)
            flag = False
            break

array.sort()

for i in array:
    print(i)